erpnext/accounts/doctype/accounts_settings/accounts_settings.json

```
@@ -376,14 +377,20 @@
    "fieldname": "auto_reconcile_payments",
    "fieldtype": "Check",
    "label": "Auto Reconcile Payments"
+  },
+  {
+   "default": "0",
```

1. Defaults should usually be consistent with current behaviour to avoid breaking changes. 
2. Singles can not sync defaults on existing sites so you should either patch it or rewrite business logic so default of `0` makes sense. i.e. `dont show taxes as table in print` (reword it)

```suggestion
   "default": "1",
```

---

Ignore. brainfart moment.